### ISSUE TITLE
Item Rolling Bugh

### ISSUE THREAD

**Issue by smudlkCZ** [Valhalla-Gaming/Tracker#6776]
I launched Valhalla launcher after few months and I wanted to play with my friend. So we create new characters and play. After several minutes we discovered this bug. When we loot a item which want to be roll it says "This item is being rolled" so we can´t loot it. Has anybody same problem or knew somebody solution I will be grateful. (Sorry my Englisch it isn´t my primary language) And we played in party.

**Comment by demonofmirkwood** [comment]
Yeah same problem for now me and my friend switched to raid group and using personal loot

**Comment by Laevoth** [comment]
Someone also posted that you can enable 'free-for-all' by converting your party to a raid, changing loot from 'group loot' to 'free-for-all' and then converting the raid back to a party. This also is still a pretty major issue with my friend and I so it would be nice if we could get a fix as soon as we can.
